src/components/ecosystems/Collections/CollectionsContainer.jsx

```diff
@@ -140,8 +140,8 @@ export default compose(
           })
       }
     },
-    renameItem: ({ activeItem }) => () => {
-      return activeItem.rename(activeItem['name'], { message: 'Renamed item' })
+    renameItem: ({ activeItem, branch }) => () => {
+      return activeItem.rename(activeItem['name'], { message: 'Renamed item', ref: branch })
```

Comment:
@tbrandle let's update this message to be consistent with the rest `[HT] Renamed item [OLD_NAME] -> [NEW_NAME]` or something like that ...